src/bootstrap-table.js

```diff
@@ -2389,6 +2389,9 @@
             return;
         }
         this.data[params.index][params.field] = params.value;
+
+        var reinit = params.reinit;
+        if (reinit===false || reinit==='false') return;
```

Comment:
This would break compatibility with current/previous behaviour. Existing **updateCell** calls without this param would not work as before. **reinit** has to have value set. undefined can't pass this condition.
Do you suggest different approach?